chef/cookbooks/hyperv/recipes/openstack_install_nova.rb

```diff
@@ -24,3 +24,14 @@
   EOH
   not_if { ::File.exists?("#{node[:openstack][:nova][:installed]}") }
 end
+
+ruby_block "ensure correct python path in shebang" do
```

Comment:
This implies more refactoring and dependency to https://github.com/crowbar/crowbar-core
The function would be to search and replace a line in a file, the search being a regexp and the replacement as string. What we need to do is to discuss if it's worth to have this function for future use and estimate the effort in a new card.